src/pickup.cpp

```diff
@@ -191,8 +191,15 @@ bool pick_one_up( item_location &loc, int quantity, bool &got_water, bool &offer
     const auto wield_check = u.can_wield( newit );
     if( newit.has_owner() &&
         newit.get_owner() != g->faction_manager_ptr->get( faction_id( "your_followers" ) ) ) {
-        if( !query_yn( "Picking up this item will be considered stealing, continue?" ) ) {
-            return false;
+        if( u.thief == THIEF_UNSET ) { // Has the player given input on if stealing is ok?
+            if( query_yn( "Picking up this item will be considered stealing, continue?" ) ) {
```

Comment:
Needs to be translatable - see "doc/TRANSLATING.md".

Reply:
Notice that I didnt add that question, but I dont mind adding _() around it. :)

Reply:
Yep, that one is my bad.